common/lib/xmodule/xmodule/services.py

```diff
@@ -0,0 +1,19 @@
+"""
+Module contains various XModule/XBlock services
```

Comment:
Oh - hm `xblock_setting_key`s are strings - kind of confusing. These are meant to be the names of the block, yeah? Why not `xblock_name`? I think I am missing something fairly key here.